Library/Homebrew/cmd/test-bot.rb

```diff
@@ -573,11 +573,16 @@ def formula(formula_name)
     def homebrew
       @category = __method__
       return if @skip_homebrew
-      test "brew", "tests"
-      test "brew", "tests", "--no-compat" if @tap
-      readall_args = ["--aliases"]
-      readall_args << "--syntax" if MacOS.version >= :mavericks
-      test "brew", "readall", *readall_args
+      if @tap
+        test "brew", "readall", @tap
```

Comment:
Sorry I forget about this part. Fixed in 4c6e764f95cbe1074731b4ff3b31a23a7aeb4a42